deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -613,7 +613,7 @@ spec:
                 - name: WATCH_NAMESPACE
                   valueFrom:
                     fieldRef:
-                      fieldPath: metadata.namespace
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
This diff looks like a transition from the value being a single NS to potentially multiple (`olm.targetNamespaces`). Will we get a single value from this annotation? It sounds like a list. Will it be usable as a value for `WATCH_NAMESPACE`?

Reply:
can we test this on a live cluster and see what we get? I think this was the change that Chandler had suggested.

Reply:
I've tested on my cluster, and for an all-namespaces case, it works as expected.

@pmorie The value should be a single NS because at 

https://github.com/openshift/kubefed-operator/blob/master/deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml#L682

you explicitly refuse to accept multi-namespace values.

Reply:
thanks @cwilkers. So you changed the CSV to take the targetNamespaces as the fieldPath for the WATCH_NAMESPACE env-var and everything works as desired?

Reply:
@abhat I just reinstalled to the single namespace case and everything lgtm